controllers/topics.controllers.js

```diff
@@ -46,3 +46,12 @@ exports.getArticleComments = (request, response, next) => {
         next(err)
     })
 }
+
+exports.postArticleComment = (request, response, next) => {
+    addComment(request.body, request.params.article_id).then((comment) => {
```

Comment:
Would like to see you have req.body/article_id saved as variables and then passed in to the function.

It's simple but makes everything more readable.